GUIs/ScriptRunnerPyQt5_GUI/scripFunc/scripWINONLY.py

```diff
@@ -327,3 +323,436 @@ def browserCONF(self):
         shutil.copytree('./configurations/Win_Mozilla/SystemExtensionsDev',
                         r'C:\Users\Michael\AppData\Roaming\Mozilla\SystemExtensionsDev')
         print('Firefox has been configured')
+
+
+class funcWINusrgru:
+    def addusr(self):
```

Comment:
Codacy found an issue: [funcWINusrgru.addusr is too complex (42) (MC0001)](https://app.codacy.com/manual/vipersniper0501/CP_Scripts2/pullRequest?prid=5823671)